src/main/java/uk/gov/hmcts/divorce/divorcecase/model/State.java

```diff
@@ -53,6 +54,20 @@
     )
     AosOverdue("AosOverdue"),
 
+    @CCD(
+        name = "Applicant 2 approved",
+        label = "# **${[CASE_REFERENCE]}** ${applicant1LastName} **&** ${applicant2LastName}\n### **${[STATE]}**\n",
+        access = {CaseAccessAdministrator.class}
```

Comment:
@linusnorton do we need CAA access on this one ?

Reply:
no, good catch